src/azure/cli/_output.py

```diff
@@ -78,6 +54,20 @@ def format_text(obj):
     except TypeError:
         return ''
 
+def format_table(obj):
+    result = obj.result
+    try:
+        if obj.simple_output_query and not obj.is_query_active:
+            if callable(obj.simple_output_query):
```

Comment:
If simple output query is just the callable, why not rename it "table_transformer" or something. The old name was always pretty vague and now it's REALLY vague considering it only does that one thing. You shouldn't need the callable check on it now since we're saying that the callable is the only supported thing.  If a dev passed a query string he should be given an error message he can debug rather than have it simply ignore him (or her). :)